### PR TITLE
#62

### PR DESCRIPTION
**Pull request by mchen5517**
updated product card and fixed home and products page to use it

### CODE REVIEW

**Review by miggitymack — APPROVED** [review]
reviewed and looks good

File: app/components/Home.jsx [hunk 1]
```diff
@@ -8,12 +8,13 @@ import ProductCard from './ProductCard'
...
     <Row>
-       <Col s={12} m={6} l={4} className='grid-example' key='1'> <ProductCard /> </Col>
-       <Col s={12} m={6} l={4} className='grid-example' key='2'> <ProductCard /> </Col>
-       <Col s={12} m={6} l={4} className='grid-example' key='3'> <ProductCard /> </Col>
-       <Col s={12} m={6} l={4} className='grid-example' key='4'> <ProductCard /> </Col>
-       <Col s={12} m={6} l={4} className='grid-example' key='5'> <ProductCard /> </Col>
-       <Col s={12} m={6} l={4} className='grid-example' key='6'> <ProductCard /> </Col>
+      { 
+        props.products && props.products.map(product => (
+          <Col s={4} className='grid-example' key={product.id}>
```
Comment (fourestfire) [inline app/components/Home.jsx]: this is great, but could you add the s12 m6 and l4 back in? that helps make it responsive.

Comment (fourestfire) [reply app/components/Home.jsx]: we should probably do that for the all products page as well

File: app/reducers/products.jsx [hunk 2]
```diff
@@ -50,8 +50,17 @@ export const fetchProducts = (searchStr, offset) => dispatch => {
...
     .then(product => dispatch(setProduct(product)))
     .catch(err => console.log(err)) // TODO: real err handling
 };
 
+// Fetch products for home page
+export const fetchHomeProducts = () => dispatch => {
+  axios.get('/api/products/homepage')
+    .then(res => res.data)
+    .then(products => dispatch(setProducts(products)))
+    .catch(err => console.log(err)) // TODO: real err handling
```
Comment (fourestfire) [inline app/reducers/products.jsx:63]: what is real error handling anyway? i don't remember what else you can do here

File: app/main.jsx [hunk 3]
```diff
@@ -31,12 +31,12 @@ const App = connect(
...
 )
 
-const RoutesComponent = ({onProductsEnter}) => (
+const RoutesComponent = ({onProductsEnter, onHomeEnter}) => (
     <Router history={browserHistory}>
       <Route path="/" component={App}>
         <IndexRedirect to="/home" />
-        <Route path="/home" component={Home} />
-        <Route path="/products" component={Products} onEnter={onProductsEnter}/>
+        <Route path="/home" component={Home} onEnter={onHomeEnter} />
```
Comment (fourestfire) [inline app/main.jsx:40]: AWESOME.

File: server/products.js [hunk 4]
```diff
@@ -13,6 +13,13 @@ module.exports = require('express').Router()
       Product.searchWithOffset(req.query.search, req.query.offset)
         .then(products => res.json(products))
         .catch(next))
+  // get up to 6 products for the front page
+  // TODO: get top bought products?
+  .get('/homepage',
+    (req,res,next) =>
+      Product.findAll({limit: 6})
```
Comment (fourestfire) [inline server/products.js:20]: cool. do you think we should use our offset vars or is that something different?

**Review by fourestfire — APPROVED** [review]

**Pull request merged**